### PR TITLE
Preliminary Results Page

### PR DESCRIPTION
**Pull request by eharley19**
This PR adds preliminary results page. Also modifies flask app to allow cross-origin requests.

### CODE REVIEW

File: frontend/13f-app/src/components/pages/searchResults.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import Results from '../results'
+
+function SearchResults () {
+  return (
+        <>
+         <Results />
+
```
Comment (TheNorthEestern) [inline frontend/13f-app/src/components/pages/searchResults.js:8]:
```suggestion
```

File: frontend/13f-app/src/components/results.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+function Results () {
+  const [results, setResults] = useState([])
+  useEffect(() => {
+    axios.get('http://localhost:5000/company/search?q=t')
+      .then(res => {
+        const companies = res.data
+        setResults(companies)
+      })
+  }, [])
```
Comment (TheNorthEestern) [inline frontend/13f-app/src/components/results.js:12]:
```suggestion

  useEffect(() => {
    axios.get('http://localhost:5000/company/search?q=t')
      .then(res => {
        const companies = res.data
        setResults(companies)
      })
  }, [])
  
```

File: python/edgar_filing_searcher/api/routes/company.py [hunk 3]
```diff
@@ -6,6 +6,11 @@
 
 company_blueprint = Blueprint('company', __name__)
 
+@company_blueprint.after_request # blueprint can also be app~~
+def after_request(response):
+    header = response.headers
+    header['Access-Control-Allow-Origin'] = '*'
```
Comment (TheNorthEestern) [inline python/edgar_filing_searcher/api/routes/company.py:12]: [Enables Cross Origin Resource Sharing](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS)

File: frontend/13f-app/src/components/pages/searchResults.js (hunk 1 above)
Comment (eharley19) [reply frontend/13f-app/src/components/pages/searchResults.js]: Edit made.

File: frontend/13f-app/src/components/results.js (hunk 2 above)
Comment (eharley19) [reply frontend/13f-app/src/components/results.js:13]: Edit made.

File: python/edgar_filing_searcher/api/routes/company.py (hunk 3 above)
Comment (eharley19) [reply python/edgar_filing_searcher/api/routes/company.py:13]: Edit made.

**Review by datdo — APPROVED** [review]

**Pull request merged**